<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 63 9pt &quot;Bahnschrift SemiBold SemiConden&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>311</x>
          <y>9</y>
          <width>640</width>
          <height>480</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>480</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1280</width>
          <height>960</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border-style: solid;
border-width: 5px;
border-color: rgb(15, 107, 255);
border-radius: 10px;</string>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>540</y>
          <width>521</width>
          <height>61</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(15, 107, 255);
font: 18pt &quot;맑은 고딕&quot;;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;로그인을 위해 QR코드를 인식해주세요&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>50</y>
          <width>640</width>
          <height>450</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>450</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-style: solid;
border-width: 5px;
border-color: rgb(15, 107, 255);
border-radius: 10px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>570</y>
          <width>300</width>
          <height>60</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>60</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(15, 107, 255);
color: rgb(255, 255, 255);
font: 18pt &quot;맑은 고딕&quot;;

</string>
        </property>
        <property name="text">
         <string>촬영</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>540</y>
          <width>544</width>
          <height>20</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Ubuntu&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>1100</x>
          <y>570</y>
          <width>100</width>
          <height>60</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(15, 107, 255);
font: 15pt &quot;맑은 고딕&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>장바구니</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>550</y>
          <width>200</width>
          <height>60</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>60</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(15, 107, 255);
border-style: solid;
border-width: 5px;
border-color: rgb(15, 107, 255);
border-radius: 10px;
color: rgb(255, 255, 255);
font: 18pt &quot;맑은 고딕&quot;;</string>
        </property>
        <property name="text">
         <string>장바구니 담기</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>60</y>
          <width>640</width>
          <height>450</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>450</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>640</width>
          <height>450</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-style: solid;
border-width: 5px;
border-color: rgb(15, 107, 255);
border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>1050</x>
          <y>550</y>
          <width>100</width>
          <height>60</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(15, 107, 255);
color: rgb(255, 255, 255);
border-style: solid;
border-width: 5px;
border-color: rgb(15, 107, 255);
border-radius: 10px;
font: 18pt &quot;맑은 고딕&quot;;</string>
        </property>
        <property name="text">
         <string>뒤로가기</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>860</x>
          <y>60</y>
          <width>300</width>
          <height>450</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>450</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>450</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
color: rgb(0, 0, 0);
border-style: solid;
border-width: 5px;
border-color: rgb(15, 107, 255);
border-radius: 10px;
font: 16pt &quot;Ubuntu&quot;;</string>
        </property>
        <property name="text">
         <string>123</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>30</y>
          <width>640</width>
          <height>480</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>480</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>640</width>
          <height>450</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-style: solid;
border-width: 5px;
border-color: rgb(15, 107, 255);
border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>사진</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>940</x>
          <y>540</y>
          <width>200</width>
          <height>60</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(15, 107, 255);
border-style: solid;
border-width: 5px;
border-color: rgb(15, 107, 255);
border-radius: 10px;
font: 15pt &quot;맑은 고딕&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>뒤로가기</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>810</x>
          <y>540</y>
          <width>96</width>
          <height>60</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(15, 107, 255);
border-style: solid;
border-width: 5px;
border-color: rgb(15, 107, 255);
border-radius: 10px;
color: rgb(255, 255, 255);
font: 15pt &quot;맑은 고딕&quot;;</string>
        </property>
        <property name="text">
         <string>로그아웃</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>540</y>
          <width>271</width>
          <height>60</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(15, 107, 255);
border-style: solid;
border-width: 5px;
border-color: rgb(15, 107, 255);
border-radius: 10px;
color: rgb(255, 255, 255);
font: 15pt &quot;맑은 고딕&quot;;</string>
        </property>
        <property name="text">
         <string>메뉴 추천</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>830</x>
          <y>100</y>
          <width>310</width>
          <height>352</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item alignment="Qt::AlignTop">
          <widget class="QLabel" name="label_6">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>200</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-style: solid;
border-width: 5px;
border-color: rgb(15, 107, 255);
border-radius: 10px;
color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>리스트 출력</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="pushButton_11">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(15, 107, 255);
border-style: solid;
border-width: 5px;
border-color: rgb(15, 107, 255);
border-radius: 10px;
font: 15pt &quot;맑은 고딕&quot;;
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Before</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_10">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(15,107, 255);
border-style: solid;
border-width: 5px;
border-color: rgb(15, 107, 255);
border-radius: 10px;
color: rgb(255, 255, 255);
font: 15pt &quot;맑은 고딕&quot;;</string>
             </property>
             <property name="text">
              <string>Next</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-style: solid;
border-width: 5px;
border-color: rgb(15, 107, 255);
border-radius: 10px;
color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>페이지</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_12">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(15, 107, 255);
border-style: solid;
border-width: 5px;
border-color: rgb(15, 107, 255);
border-radius: 10px;
font: 15pt &quot;맑은 고딕&quot;;
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>장바구니에서 제거</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_5">
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>70</y>
          <width>351</width>
          <height>281</height>
         </rect>
        </property>
        <property name="text">
         <string>상품 이미지</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>90</y>
          <width>181</width>
          <height>151</height>
         </rect>
        </property>
        <property name="text">
         <string>성분표 출력력</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_8">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>320</y>
          <width>141</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>장바구니에 제거</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_9">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>440</y>
          <width>91</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>뒤로가기</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_6">
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>511</width>
          <height>411</height>
         </rect>
        </property>
        <property name="text">
         <string>요리 이미지</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>80</y>
          <width>631</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 18pt &quot;Ubuntu&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>요리 이름</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>180</y>
          <width>631</width>
          <height>261</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>요리 재료</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>480</y>
          <width>611</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>요리 링크</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_7">
       <widget class="QPushButton" name="pushButton_13">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>510</y>
          <width>641</width>
          <height>100</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(2, 78, 255);
font: 24pt &quot;Ubuntu&quot;;</string>
        </property>
        <property name="text">
         <string>시작하기</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>311</x>
          <y>255</y>
          <width>641</width>
          <height>121</height>
         </rect>
        </property>
        <property name="text">
         <string>Wlcome to WSIE</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>311</x>
          <y>9</y>
          <width>640</width>
          <height>240</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>240</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>640</width>
          <height>240</height>
         </size>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
